apps/web/src/features/calendar/calendar.css

```
@@ -0,0 +1,473 @@
+	.fc-timegrid-slot-label-cushion {
+	visibility: hidden;
+}
+
+.calendar-view .fc .fc-timegrid-now-indicator-line {
+	--calendar-now-arrow-bottom: 50%;
+	--calendar-now-arrow-top: 50%;
+	--calendar-now-axis-height: 0.75rem;
+	--calendar-now-axis-radius: 0.125rem;
+	--calendar-now-axis-width: 0.625rem;
+	--calendar-now-expanded: 0;
+	border: 0;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)